### PR TITLE
Properly pass xml report flags to ATen tests in Caffe2 builds

### PR DESCRIPTION
**Pull request by pjh5**
ATen uses Catch2 as test framework while Caffe2 uses Google Test

### CODE REVIEW

**Review by yinghai — COMMENTED** [review]
Makes sense. But do we care about those tests from aten? How does Pytorch ci trigger those tests?

**Comment by pjh5** [comment]
Hopefully the tests will get coverage on pytorch-without-caffe2 builds. This only affects when both ATen and Caffe2 are built together

File: .jenkins/caffe2/test.sh [hunk 1]
```diff
@@ -53,7 +53,12 @@ cd ${INSTALL_PREFIX}
 
 # C++ tests
 echo "Running C++ tests.."
-for test in $INSTALL_PREFIX/test/*; do
+for test in $(find $INSTALL_PREFIX/test/); do
+  # Skip directories
+  if [ $(dirname "${test}") == 'aten' ]; then
+    "$test" -r=xml -o "${test}.xml"
```
Comment (bddppq) [inline .jenkins/caffe2/test.sh]: Could you double check in our job-dsl repo which directories does it collect xml report files?

**Review by yinghai — COMMENTED** [review]
```
16:31:26 + /usr/local/caffe2/test/ --gtest_output=xml:/var/lib/jenkins/workspace/caffe2_tests/cpp/test.xml
16:31:26 ./.jenkins/caffe2/test.sh: line 69: /usr/local/caffe2/test/: Is a directory
```

**Comment by yinghai** [comment]
In addition, could you rebase against master to pick up @orionr's fix?

**Comment by yinghai** [comment]
FTR, `aten/integer_divider_test` does not hang. It is just very slow....

**Review by bddppq — APPROVED** [review]

**Pull request merged**